test/bdd/fixtures/vc-rest/docker-compose.yml

```diff
@@ -43,6 +45,8 @@ services:
       - UNIVERSAL_RESOLVER_HOST_URL=http://uni-resolver-web:8080/1.0/identifiers
       - DATABASE_TYPE=couchdb
       - DATABASE_URL=couchdb.example.com:5984
+      - KMSSECRETS_DATABASE_TYPE=couchdb
```

Comment:
@DRK3 we need to add here also:https://github.com/trustbloc/edge-service/blob/master/test/bdd/fixtures/vc-rest/docker-compose.yml#L58

Reply:
done